test/full/component_validation_test.go

```diff
@@ -670,6 +672,33 @@ func getMinimalConfig(componentName string, wombatType string) string {
     urls: ["tcp://localhost:1883"]
     topic: "test/topic"
     client_id: "test-client"`
+	case "sql_raw":
+		if wombatType == "input" {
+			return `
+    driver: "clickhouse"
+    dsn: clickhouse://username:password@host1:9000,host2:9000/database?dial_timeout=200ms&max_execution_time=60
+    query: SELECT * FROM footable;
+    auto_replay_nacks: true`
+		}
+		return `
+    driver: "clickhouse"
+    dsn: clickhouse://username:password@host1:9000,host2:9000/database?dial_timeout=200ms&max_execution_time=60
+    max_in_flight: 64`
+	case "snowflake_put":
+		return `
+    account: ""
```

Comment:
waaaay to many fields that are required with default values

Reply:
trimmed these down to the bare minimum